.github/workflows/pr.yml

```diff
@@ -63,13 +65,18 @@ jobs:
       with:
         driver: docker
         driver-opts: network=host
-    - name: Login to Docker Hub
+    - if: ${{ env.dockerhub_username != '' }}
```

Comment:
I presume this is because you can't use github secrets in conditionals?

I think yes, but just double-checking the reason for setting this as an env variable.